packages/manager/modules/pci/src/projects/project/kubernetes/kubernetes.routing.js

```diff
@@ -53,6 +53,29 @@ export default /* @ngInject */ ($stateProvider) => {
         return promise;
       },
 
+      getKubeApiErrorId: /* @ngInject */ () => (error) => {
+        try {
+          const errorMessage = get(error, 'data.message');
```

Comment:
```suggestion
          const errorMessage = errror.data?.message;
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining